package.json

```diff
@@ -30,7 +30,8 @@
   },
   "homepage": "https://github.com/legomushroom/mojs",
   "dependencies": {
-    "babel-runtime": "^6.5.0"
+    "babel-runtime": "^6.5.0",
+    "gulp-babel": "^6.1.2"
```

Comment:
When I tried to run `gulp` to build, it complained about the missing package. I assumed most contributors here have it installed for other uses before here, but I didn't. It stopped complaining and worked flawlessly afterwards. (Also, it makes sense, as the files need to go through babel in order to compile to legacy JS)